std/algorithm/comparison.d

```diff
@@ -475,10 +475,19 @@ unittest
                            )());
 }
 
-/**
-Returns $(D val), if it is between $(D lower) and $(D upper).
-Otherwise returns the nearest of the two. Equivalent to $(D max(lower,
-min(upper,val))).
+/** Clamps a value into bounds.
+
+This functions is equivalent to $(D max(lower, min(upper,val))).
+
+Params:
+    val = The value to clamp.
```

Comment:
Hmm, looks like we need to write `_clamp` here as well, since it coincides with the function name and ddoc will auto-highlight it.